core/runner.py

```diff
@@ -103,23 +97,42 @@ def run_job(self, job):
                     job.serialized_model,
                     job.framework_type
                 )
-                weights_in_bytes = serialize_weights(initial_weights)
-                # TODO: Send (weights_in_bytes) to all nodes/aggregator/developer
-                # through P2P.
+                # weights_in_bytes = serialize_weights(initial_weights)
                 # logging.info(initial_weights)
                 results = DMLResult(
                     status='successful',
-                    job_type=job.job_type,
+                    job=job,
                     results={
-                        'initial_weights': initial_weights,
+                        'weights': initial_weights,
                     },
                     error_message="",
                 )
+            elif job.job_type == 'average':
+                raise Exception("averaging is not yet supported!")
```

Comment:
Did you mean to still leave this here?

Reply:
Yes. The upcoming PR will implement this. Same with "communicate".